options/options.js

```diff
@@ -144,3 +160,53 @@ function updateState() {
     option.value = newValue;
   });
 }
+
+function moveUp(id) {
+  // move the element up
+  var id_temp = (id.toString()).replace("-move-up","");
+  document.getElementById(id_temp+"-priority").value = parseInt(document.getElementById(id_temp+"-priority").value)-1;
+  // call "onchange" event (forced)
+  if ("createEvent" in document) {
+    var evt = document.createEvent("HTMLEvents");
+    evt.initEvent("change", false, true);
+    document.getElementById(id_temp+"-priority").dispatchEvent(evt);
+  }
+  else
+    document.getElementById(id_temp+"-priority").fireEvent("onchange");
+  setFocusRowTemp(parseInt(document.getElementById(id_temp+"-priority").value)-1);
+  document.getElementById(id_temp+"-priority").focus();
+}
+
+function moveDown(id) {
+  // move the element down
+  var id_temp = (id.toString()).replace("-move-down","");
+  document.getElementById(id_temp+"-priority").value = parseInt(document.getElementById(id_temp+"-priority").value)+1;
+  // call "onchange" event (forced)
+  if ("createEvent" in document) {
+    var evt = document.createEvent("HTMLEvents");
+    evt.initEvent("change", false, true);
+    document.getElementById(id_temp+"-priority").dispatchEvent(evt);
+  }
+  else
+    document.getElementById(id_temp+"-priority").fireEvent("onchange");
+  setFocusRowTemp(parseInt(document.getElementById(id_temp+"-priority").value)-1);
+  document.getElementById(id_temp+"-priority").focus();
+}
+
+function setFocusRowTemp(index) {
+  // set "focus" on row modified, before clear eventually wrong "class"
+  clearFocusRowTemp();
+  document.getElementsByClassName("service")[index].classList.add('row_focus');
+  setTimeout(function() {
+    document.getElementsByClassName("service")[index].classList.remove('row_focus')
+  }, 1000);
+}
+
+function clearFocusRowTemp()
+{
+  document.querySelectorAll('div').forEach(function(item) {
+    if (item.classList.contains("service")) {
```

Comment:
I don't think that this if is required, if the method is clear we want to remove the class to any item.

Reply:
It cause some bugs ... if you press fast on some buttons (differents) -> they stay "focussed"